source/assets/scripts/main.js

```diff
@@ -0,0 +1,101 @@
+// main.js
+
+const $ = (selector) => document.querySelector(selector);
+
+
+const featureRecipes = [
+  'https://introweb.tech/assets/json/ghostCookies.json',
+  'https://introweb.tech/assets/json/birthdayCake.json',
+  'https://introweb.tech/assets/json/chocolateChip.json',
+  'assets/recipes/meatball.json',
+  'assets/recipes/quickEasyPaste.json',
+  'assets/recipes/pumpkinMuffins.json'
+];
+
+let myRecipeArray = JSON.parse(localStorage.getItem('myRecipeArray'));
+
+
+const feaRecipeData = {};
+const favRecipeData = {};
+
+window.addEventListener('DOMContentLoaded', init);
+
+
+async function init() {
+  let fetchSuccessful = await fetchRecipes();
+  if (!fetchSuccessful) {
+    console.log('Recipe fetch unsuccessful');
+    return;
+  };
+
+  let addFeaCardSuccessful = await createFeaRecipeCards();
+  if (!addFeaCardSuccessful) {
+    console.log('Add recipe unsuccessful');
+    return;
+  };
+
+  let addFavCardSuccessful = await createFavRecipeCards();
+  if (!addFavCardSuccessful) {
+    console.log('Add recipe unsuccessful');
+    return;
+  };
+
+  main();
+}
+
+function main() {
+  let addNewCard = document.createElement('new-card');
+  $('#my-list').appendChild(addNewCard);
+  let addEmptyCard = document.createElement('empty-card');
+  $('#my-list').appendChild(addEmptyCard);
+  addNewCard = document.createElement('empty-card');
+  $('#my-list').appendChild(addNewCard);
+}
+
+async function fetchRecipes() {
+  return new Promise((resolve, reject) => {
+    for (let i = 0; i < featureRecipes.length; i++) {
+      fetch(featureRecipes[i])
+        .then(response => {
+          if (!response.ok) {
+            reject(false);
+          }
+          else {
+            return response.json();
+          }
+        })
+        .catch(e => reject(false))
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/cse110-fa21-group16/cse110-fa21-group16/pullRequest?prid=8393580)
```suggestion
        .catch((e) => reject(false))
```